data/scripts/debug.inc

```diff
@@ -87,14 +87,26 @@ Debug_FlagsNotSetBattleConfigMessage_Text:
 	.string "Feature unavailable!\n"
 	.string "Please define a usable flag in:\l"
 	.string "'include/config/battle.h'!$"
-
```

Comment:
Is there a reason this newline was removed?

Reply:
fca1d479750551aa9384927a93684fc04a40ff46